js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx

```diff
@@ -76,6 +82,11 @@ export const AttributeGeneralSettings = () => {
     form.setValue("hasRequiredScopes", hasRequiredScopes);
   }
 
+  function setMultivalued(multivalued: boolean) {
+    console.log(multivalued);
```

Comment:
I guess you used `console.log` during development, but should be likely removed from this PR?